src/style.css

```diff
@@ -0,0 +1,109 @@
+body {
+  margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  max-width: 200px;
+  padding: 30px 14px 14px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 5px 45px;
+}
+
+.product__description {
+  font-size: 12px;
+  line-height: 15px;
+  font-weight: 500;
+  margin: 0 0 5px;
+}
+
+.product__code {
+  font-size: 11px;
+  line-height: 13px;
+  margin: 0 0 15px;
+}
+
+.product__rating {
+  height: 15px;
+  margin: 0 0 20px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.product__stars {
+  display: flex;
+  align-items: center;
+}
+
+.star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--5 .star,
+.stars--1 .star:first-child,
+.stars--2 .star:nth-child(-n+2),
+.stars--3 .star:nth-child(-n+3),
+.stars--4 .star:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.product__feedbacks {
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.product__value {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+.product__caption {
+  margin: 0;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.product__price {
+  margin: 0 0 20px;
+  font-weight: 900;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.product__button {
+  width: 170px;
+  height: 40px;
+  color: #fff;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 14px;
+  font-weight: bold;
+  line-height: 40px;
+  background-color: #00acdc;
+  text-align: center;
+  border-radius: 5px;
+}
+
+.product__button:hover {
+  box-sizing: border-box;
+  color: #00acdc;
+  background-color: #fff;
+  border: 1px solid #00acdc;
```

Comment:
don't add border to hovered button. Add it to initial state, on hover just change it's color